test/Splitter.t.sol

```diff
@@ -175,17 +174,17 @@ contract SplitterTest is DssTest {
         assertEq(dss.chainlog.getAddress("MCD_FARM_USDS"), address(farm));
 
         // Add initial liquidity if needed
-        (uint256 reserveDai, ) = UniswapV2Library.getReserves(UNIV2_FACTORY, DAI, MKR);
-        uint256 minimalDaiReserve = 280_000 * WAD;
-        if (reserveDai < minimalDaiReserve) {
+        (uint256 reserveUsds, ) = UniswapV2Library.getReserves(UNIV2_FACTORY, USDS, SKY);
+        uint256 minimalUsdsReserve = 280_000 * WAD;
+        if (reserveUsds < minimalUsdsReserve) {
```

Comment:
Maybe more clear if the next line uses a realistic SKY price (e.g. 0.06 ether)?

Reply:
Yes I was wondering what to do on prices as the base pip is the MKR one which is confusing. Maybe I should just use any other random as the only important thing is actually the contract functionality.

Reply:
Yes or if it's not too much change we could also use the OracleWrapper with the PIP_MKR.

Reply:
Yeah at some point started to do that as well, but then I saw the flappers test use a mock and the oracle wrapper test PIP_ETH that wasn't convinced what path to take. Will give a bit more of thought to that to try to standardise a bit.